woodwork/serialize.py

```diff
@@ -223,3 +224,43 @@ def save_orc_file(dataframe, filepath):
             df[c] = df[c].astype("string")
     pa_table = Table.from_pandas(df, preserve_index=False)
     orc.write_table(pa_table, filepath)
+
+
+def _save_parquet_file(dataframe, path, filename, profile_name):
+    """Write a Woodwork dataframe to disk, saving typing information in
+    the parquet file metadata."""
+    import pyarrow as pa
+    import pyarrow.parquet as pq
+
+    ww_typing = typing_info_to_dict(dataframe)
+
+    if _is_dask_dataframe(dataframe):
+        ww_metadata = {METADATA_KEY: json.dumps(ww_typing)}
+        dataframe.to_parquet(path, custom_metadata=ww_metadata)
+    else:
+        if filename is None:
+            filename = "data.parquet"
+        table = pa.Table.from_pandas(dataframe)
+        table_meta = table.schema.metadata
+        combined_metadata = {
+            METADATA_KEY.encode(): json.dumps(ww_typing).encode(),
+            **table_meta,
+        }
+        table = table.replace_schema_metadata(combined_metadata)
```

Comment:
Something we might want to do here is this:
```suggestion
        table_meta = table.schema.metadata
        table_meta[METADATA_KEY.encode()] = json.dumps(ww_typing).encode()
        table = table.replace_schema_metadata(table_meta)
```
The advantage of doing it this way is that we would always overwrite the `METADATA_KEY` key which I believe is what's intended here.  If that's not the intention, the old approach is also fine I think.

Reply:
Yeah, that's fair and probably a little cleaner. I mostly copied this code from another example on how to update parquet metadata and didn't think through that detail enough.